server/models/user.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique:true
+    },
+    username: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique:true
+    },
+    bio: {
+      type: DataTypes.TEXT
+    },
+    imgUrl: {
+      type: DataTypes.STRING
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    isAdmin: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
```

Comment:
This is not the full user model agreed where are the associations or relationships

Reply:
I did not implement associations and relationships because you only approved user model. You said the other models might change, that was why I did not create them

Reply:
the task was to implement sequelize you don't need to implement a model to test the connection